spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -152,6 +154,15 @@ public SparkContext getSparkContext() {
     }
   }
 
+  public JavaSparkContext getJavaSparkContext() {
+    synchronized (sharedInterpreterLock) {
+      if (jsc == null) {
+        jsc = JavaSparkContext.fromSparkContext(sc);
```

Comment:
this needs to be cleared along with `sc`, correct?
https://github.com/vipul1409/zeppelin/blob/cebf9701d4be708af27eed3eb786f8bc3d2bf628/spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java#L1435